coordinator-server/event-client/actionSniffer.js

```diff
@@ -106,12 +133,7 @@ class ActionSniffer {
                 // let logger = (args) => console.log(`My args are ${args}`);
                 // // throttle: call the logger at most once every two seconds
                 // let throttledLogger = throttle(logger, 2000);
-                if (isDebounced) {
-                    window.addEventListener(eventName, this.debounce(listener, 30), capture);
```

Comment:
shouldn't we debounce some events still? because I think it might hurt performance of the other browser, because it will get 50 scroll/mouseover events a second and will execute each and every one of them.
did you see any performance decrease? if you did, was it noticeable?